From 9a55b996651fdfbc40e202cb4dd725dce624ca91 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Fri, 24 Jun 2016 20:02:01 +0200
Subject: qt5: bump version to 5.6.1-1

Removed obsolete patches:

  - qt5base/0001-Disable-c-standard-compiler-flags-for-the-host-build.patch (upstream committed [1])
  - qt5/qt5base/0007-build-with-explicitlib-after-all.patch (upstream committed [2])
  - qt5tools/0001-Disable-qdoc-needs-qtdeclarative.patch (fixed upstream by [3])
  - qt5webkit/0004-Fix-linking-with-libpthread.patch (upstream committed [4])

[1] http://code.qt.io/cgit/qt/qtbase.git/commit/?id=e69e69519661954716d59bfa5bbd0626515cfda9
[2] http://code.qt.io/cgit/qt/qtbase.git/commit/?id=523c7e3fd55c853dd424d57f28e225d57439cf89
[3] http://code.qt.io/cgit/qt/qttools.git/commit/?id=2b262fad86ef38a5fa692b4c73e6ec26a5d45a5f
[4] http://code.qt.io/cgit/qt/qtwebkit.git/commit/?id=5f359baacdf92fabcece83f0a2b30f74c7c02a3c

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 package/qt5/qt5.mk                                 |  2 +-
 package/qt5/qt53d/qt53d.hash                       |  4 +-
 ...tandard-compiler-flags-for-the-host-build.patch | 44 -------------------
 .../0007-build-with-explicitlib-after-all.patch    | 51 ----------------------
 package/qt5/qt5base/qt5base.hash                   |  5 +--
 package/qt5/qt5base/qt5base.mk                     |  1 -
 package/qt5/qt5canvas3d/qt5canvas3d.hash           |  4 +-
 package/qt5/qt5connectivity/qt5connectivity.hash   |  4 +-
 package/qt5/qt5declarative/qt5declarative.hash     |  4 +-
 package/qt5/qt5enginio/qt5enginio.hash             |  4 +-
 .../qt5graphicaleffects/qt5graphicaleffects.hash   |  4 +-
 package/qt5/qt5imageformats/qt5imageformats.hash   |  4 +-
 package/qt5/qt5location/qt5location.hash           |  4 +-
 package/qt5/qt5multimedia/qt5multimedia.hash       |  4 +-
 package/qt5/qt5quickcontrols/qt5quickcontrols.hash |  4 +-
 .../qt5/qt5quickcontrols2/qt5quickcontrols2.hash   |  4 +-
 package/qt5/qt5script/qt5script.hash               |  4 +-
 package/qt5/qt5sensors/qt5sensors.hash             |  4 +-
 package/qt5/qt5serialbus/qt5serialbus.hash         |  4 +-
 package/qt5/qt5serialport/qt5serialport.hash       |  4 +-
 package/qt5/qt5svg/qt5svg.hash                     |  4 +-
 .../0001-Disable-qdoc-needs-qtdeclarative.patch    | 30 -------------
 package/qt5/qt5tools/qt5tools.hash                 |  4 +-
 package/qt5/qt5webchannel/qt5webchannel.hash       |  4 +-
 .../0004-Fix-linking-with-libpthread.patch         | 34 ---------------
 package/qt5/qt5webkit/qt5webkit.hash               |  2 +-
 package/qt5/qt5webkit/qt5webkit.mk                 |  2 +-
 package/qt5/qt5websockets/qt5websockets.hash       |  4 +-
 package/qt5/qt5x11extras/qt5x11extras.hash         |  4 +-
 package/qt5/qt5xmlpatterns/qt5xmlpatterns.hash     |  4 +-
 30 files changed, 47 insertions(+), 208 deletions(-)
 delete mode 100644 package/qt5/qt5base/0001-Disable-c-standard-compiler-flags-for-the-host-build.patch
 delete mode 100644 package/qt5/qt5base/0007-build-with-explicitlib-after-all.patch
 delete mode 100644 package/qt5/qt5tools/0001-Disable-qdoc-needs-qtdeclarative.patch
 delete mode 100644 package/qt5/qt5webkit/0004-Fix-linking-with-libpthread.patch

diff --git a/package/qt5/qt5.mk b/package/qt5/qt5.mk
index e43795f..04bad81 100644
--- a/package/qt5/qt5.mk
+++ b/package/qt5/qt5.mk
@@ -1,5 +1,5 @@
 QT5_VERSION_MAJOR = 5.6
-QT5_VERSION = $(QT5_VERSION_MAJOR).0
+QT5_VERSION = $(QT5_VERSION_MAJOR).1-1
 QT5_SITE = http://download.qt.io/official_releases/qt/$(QT5_VERSION_MAJOR)/$(QT5_VERSION)/submodules
 include $(sort $(wildcard package/qt5/*/*.mk))
 
diff --git a/package/qt5/qt53d/qt53d.hash b/package/qt5/qt53d/qt53d.hash
index 8d9a3d6..fbe782d 100644
--- a/package/qt5/qt53d/qt53d.hash
+++ b/package/qt5/qt53d/qt53d.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qt3d-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 b961789daecebed030ad36568301d1fe6c1ad97e9a2726fad5bd96ad603f439d qt3d-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qt3d-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 6e71b6925a39056c18920350ea94e5a6b45a54f27ea2e04905e4b2acb464b7db qt3d-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5base/0001-Disable-c-standard-compiler-flags-for-the-host-build.patch b/package/qt5/qt5base/0001-Disable-c-standard-compiler-flags-for-the-host-build.patch
deleted file mode 100644
index 54e4db8..0000000
--- a/package/qt5/qt5base/0001-Disable-c-standard-compiler-flags-for-the-host-build.patch
+++ /dev/null
@@ -1,44 +0,0 @@
-From e69e69519661954716d59bfa5bbd0626515cfda9 Mon Sep 17 00:00:00 2001
-From: Peter Seiderer <ps.report@gmx.net>
-Date: Thu, 3 Mar 2016 15:17:31 +0100
-Subject: [PATCH] Disable c++ standard compiler flags for the host build
-MIME-Version: 1.0
-Content-Type: text/plain; charset=UTF-8
-Content-Transfer-Encoding: 8bit
-
-There is no test for c++ standard support for the host build
-(only for the target compiler/build) which leads to trouble
-in some cross compiling environments (old host compiler, new
-cross compiler):
-
-  g++: error: unrecognized command line option ‘-std=c++1z’
-
-So disable c++ standard compiler flags unconditionally for host builds.
-
-Task-number: QTBUG-51644
-Change-Id: Ifb3042e125fe199a7e081740d1171d26ccacf0c5
-Reviewed-by: Oswald Buddenhagen <oswald.buddenhagen@theqtcompany.com>
-Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
----
- mkspecs/features/default_post.prf | 5 ++++-
- 1 file changed, 4 insertions(+), 1 deletion(-)
-
-diff --git a/mkspecs/features/default_post.prf b/mkspecs/features/default_post.prf
-index cd8d885..561c8f4 100644
---- a/mkspecs/features/default_post.prf
-+++ b/mkspecs/features/default_post.prf
-@@ -95,7 +95,10 @@ breakpad {
-     !isEmpty(QMAKE_STRIP):QMAKE_POST_LINK = $$QMAKE_POST_LINK$$escape_expand(\\n\\t)$$quote($$QMAKE_STRIP $$DEBUGFILENAME)
- }
- 
--c++11|c++14|c++1z {
-+# Disable special compiler flags for host builds (needs to be changed for 5.7
-+# to fall back to c++11 because since 5.7 c++11 is required everywhere,
-+# including host builds).
-+if(!host_build|!cross_compile):if(c++11|c++14|c++1z) {
-     c++1z:       cxxstd = CXX1Z
-     else: c++14: cxxstd = CXX14
-     else:        cxxstd = CXX11
--- 
-2.1.4
-
diff --git a/package/qt5/qt5base/0007-build-with-explicitlib-after-all.patch b/package/qt5/qt5base/0007-build-with-explicitlib-after-all.patch
deleted file mode 100644
index a878ba6..0000000
--- a/package/qt5/qt5base/0007-build-with-explicitlib-after-all.patch
+++ /dev/null
@@ -1,51 +0,0 @@
-From 523c7e3fd55c853dd424d57f28e225d57439cf89 Mon Sep 17 00:00:00 2001
-From: Oswald Buddenhagen <oswald.buddenhagen@theqtcompany.com>
-Date: Thu, 3 Mar 2016 14:12:16 +0100
-Subject: [PATCH] build with explicitlib after all
-
-unlike speculated in 2fe363514, this is not a workaround at all: it
-causes that libraries' public link interfaces (LIBS) are exported in the
-first place. unlike with staticlib, this does not export LIBS_PRIVATE,
-so it wouldn't even be a particularly effective workaround for rpath
-brokenness anyway.
-
-the problem was pretty well hidden by the qt module system, which at the
-level of libraries is pretty redundant with the .prl file handling,
-which shows just how stupid the whole "design" is.
-
-unlike before, we now enable explicitlib for all libraries, not just qt
-modules - we enable create_prl for all of them as well, after all.
-
-an immediate effect of this change is that it fixes linking on RaspPI:
-the qtcore headers make the user code require linking libatomic, so we
-must add it to our public link interface.
-
-Task-number: QTBUG-51621
-Change-Id: I5742c88694db8e8a9b79d17222dc6df2b38e5ab2
-Reviewed-by: Joerg Bornemann <joerg.bornemann@theqtcompany.com>
-Reviewed-by: Allan Sandfeld Jensen <allan.jensen@theqtcompany.com>
-
-Upstream: https://codereview.qt-project.org/gitweb?p=qt%2Fqtbase.git;a=commit;h=523c7e3fd55c853dd424d57f28e225d57439cf89
-Signed-off: Peter Seiderer <ps.report@gmx.net>
----
- mkspecs/features/qt_build_config.prf | 4 ++++
- 1 file changed, 4 insertions(+)
-
-diff --git a/mkspecs/features/qt_build_config.prf b/mkspecs/features/qt_build_config.prf
-index 518fd93..b3081b9 100644
---- a/mkspecs/features/qt_build_config.prf
-+++ b/mkspecs/features/qt_build_config.prf
-@@ -72,6 +72,10 @@ CONFIG += \
-     # However, testcases should be still built with exceptions.
-     exceptions_off testcase_exceptions
- 
-+# Under Windows, this is neither necessary (transitive deps are automatically
-+# resolved), nor functional (.res files end up in .prl files and break things).
-+unix: CONFIG += explicitlib
-+
- 
- defineTest(qtBuildPart) {
-     bp = $$eval($$upper($$section(_QMAKE_CONF_, /, -2, -2))_BUILD_PARTS)
--- 
-2.7.4
-
diff --git a/package/qt5/qt5base/qt5base.hash b/package/qt5/qt5base/qt5base.hash
index 3ebc56e..2df165b 100644
--- a/package/qt5/qt5base/qt5base.hash
+++ b/package/qt5/qt5base/qt5base.hash
@@ -1,3 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtbase-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 6efa8a5c559e92b2e526d48034e858023d5fd3c39115ac1bfd3bb65834dbd67a qtbase-opensource-src-5.6.0.tar.xz
-sha256 e03740e20b8dbf52d163d8a4a37125e7c55b59a9a5e7cf7e89a08e07286b6135	b8f98d956501dfa4ce03a137f15d404930a56066.patch
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtbase-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 329678347ec5ebb404225345300a8deb1e7c991322a4c50584be550c69be7c39 qtbase-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5base/qt5base.mk b/package/qt5/qt5base/qt5base.mk
index 6378b5a..783cf3c 100644
--- a/package/qt5/qt5base/qt5base.mk
+++ b/package/qt5/qt5base/qt5base.mk
@@ -7,7 +7,6 @@
 QT5BASE_VERSION = $(QT5_VERSION)
 QT5BASE_SITE = $(QT5_SITE)
 QT5BASE_SOURCE = qtbase-opensource-src-$(QT5BASE_VERSION).tar.xz
-QT5BASE_PATCH = https://github.com/qtproject/qtbase/commit/b8f98d956501dfa4ce03a137f15d404930a56066.patch
 
 QT5BASE_DEPENDENCIES = host-pkgconf zlib pcre
 QT5BASE_INSTALL_STAGING = YES
diff --git a/package/qt5/qt5canvas3d/qt5canvas3d.hash b/package/qt5/qt5canvas3d/qt5canvas3d.hash
index 57ae4d7..99d4ff6 100644
--- a/package/qt5/qt5canvas3d/qt5canvas3d.hash
+++ b/package/qt5/qt5canvas3d/qt5canvas3d.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtcanvas3d-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 66add59e826a0161f4a4dc3ec0b44c17fad1451390b4f7c67af23ee7429d9ecf qtcanvas3d-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtcanvas3d-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 84d02cc051eeda6edb8a4da00d9a35869a447f7beb04d3c39cd375a3793f228c qtcanvas3d-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5connectivity/qt5connectivity.hash b/package/qt5/qt5connectivity/qt5connectivity.hash
index a8676a9..3dab371 100644
--- a/package/qt5/qt5connectivity/qt5connectivity.hash
+++ b/package/qt5/qt5connectivity/qt5connectivity.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtconnectivity-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 0c5cb0c100006759b6954a36e7dc66f8f1ac2b61b3f639152cf6ecb8d48a40eb qtconnectivity-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtconnectivity-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 f1d714b8f359df069b9e94e72719bcb9f5340212d9ce223453b7100c6ed7266b qtconnectivity-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5declarative/qt5declarative.hash b/package/qt5/qt5declarative/qt5declarative.hash
index 830e18f..15c0c46 100644
--- a/package/qt5/qt5declarative/qt5declarative.hash
+++ b/package/qt5/qt5declarative/qt5declarative.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtdeclarative-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 8c55f053f0e348577b56da541af74d02d0f2b61c9a6c15152b03dad03dfde04c qtdeclarative-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtdeclarative-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 1cdf60b52509972ac001f5402a42d9b144b344010ec6e34f528f7dfc6be98f24 qtdeclarative-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5enginio/qt5enginio.hash b/package/qt5/qt5enginio/qt5enginio.hash
index 3f66ee0..2105aba 100644
--- a/package/qt5/qt5enginio/qt5enginio.hash
+++ b/package/qt5/qt5enginio/qt5enginio.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtenginio-opensource-src-1.6.0.tar.xz.mirrorlist
-sha256 627ddcfbbfc3ec1a83c9dbb5f24287b5cd6cb5d3b9d09af4d1c444c6ac147f0c qtenginio-opensource-src-1.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtenginio-opensource-src-1.6.1.tar.xz.mirrorlist
+sha256 0e14d5dfcae3f0b44c52751b20a8fc5f9e2c0d53aa1025ffbabbfde43fcc1a9e qtenginio-opensource-src-1.6.1.tar.xz
diff --git a/package/qt5/qt5graphicaleffects/qt5graphicaleffects.hash b/package/qt5/qt5graphicaleffects/qt5graphicaleffects.hash
index 847458e..7e952b4 100644
--- a/package/qt5/qt5graphicaleffects/qt5graphicaleffects.hash
+++ b/package/qt5/qt5graphicaleffects/qt5graphicaleffects.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtgraphicaleffects-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 01e911fdcf85a13b927cba341d15a0baeead3eba85c4532b1b45bb5c334416e8 qtgraphicaleffects-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtgraphicaleffects-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 2b9cecf8eb3e12587788fa320119cf8d7b84775211cc1d78334d27e50040c014 qtgraphicaleffects-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5imageformats/qt5imageformats.hash b/package/qt5/qt5imageformats/qt5imageformats.hash
index 05b8cbc..e21fb17 100644
--- a/package/qt5/qt5imageformats/qt5imageformats.hash
+++ b/package/qt5/qt5imageformats/qt5imageformats.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtimageformats-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 2c854275a689a513ba24f4266cc6017d76875336671c2c8801b4b7289081bada qtimageformats-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtimageformats-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 cc45e481394dcc4eb4e565969c9bb50b87b1da23f006bf849a5f8daa375328dd qtimageformats-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5location/qt5location.hash b/package/qt5/qt5location/qt5location.hash
index e078ec5..099519f 100644
--- a/package/qt5/qt5location/qt5location.hash
+++ b/package/qt5/qt5location/qt5location.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtlocation-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 360e1519d0fcafe4f86923d224e76d56dd785dfb0e1a19fd2e2b0016799653c9 qtlocation-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtlocation-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 c9cc21ba3dd3f13376a175b092256403589a52c7851603dbfe1e15d5d9bac457 qtlocation-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5multimedia/qt5multimedia.hash b/package/qt5/qt5multimedia/qt5multimedia.hash
index f988666..e2e5a2d 100644
--- a/package/qt5/qt5multimedia/qt5multimedia.hash
+++ b/package/qt5/qt5multimedia/qt5multimedia.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtmultimedia-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 103f99d6cd266f5c4485546a75ef0c6ee7e88dc901a0be21447cf89159370686 qtmultimedia-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtmultimedia-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 4e28dcc696a9996b03a124b0f56fc86646bc2e9d29d383dc923b926040774e5d qtmultimedia-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5quickcontrols/qt5quickcontrols.hash b/package/qt5/qt5quickcontrols/qt5quickcontrols.hash
index 409efde..f996263 100644
--- a/package/qt5/qt5quickcontrols/qt5quickcontrols.hash
+++ b/package/qt5/qt5quickcontrols/qt5quickcontrols.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtquickcontrols-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 ec0896792f2a08d109ab3791aa4e47747aab22ebfad281005c4bf8f26f9f788b qtquickcontrols-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtquickcontrols-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 011047af8bc2242fa9f7e7860818ca16a62a54e06b27a6b518919f4c08725f32 qtquickcontrols-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5quickcontrols2/qt5quickcontrols2.hash b/package/qt5/qt5quickcontrols2/qt5quickcontrols2.hash
index aaf8fcf..798a8ab 100644
--- a/package/qt5/qt5quickcontrols2/qt5quickcontrols2.hash
+++ b/package/qt5/qt5quickcontrols2/qt5quickcontrols2.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtquickcontrols2-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 59ec6ea2282931bc0d0748b3979a52e1a322e7ef8d1e5490b8a34931e8b9fee0 qtquickcontrols2-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtquickcontrols2-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 4c915939f659bb5f7dc381eb011809b6030c717269d9d260fca36f8d5917ca71 qtquickcontrols2-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5script/qt5script.hash b/package/qt5/qt5script/qt5script.hash
index cc32efc..4b61d1b 100644
--- a/package/qt5/qt5script/qt5script.hash
+++ b/package/qt5/qt5script/qt5script.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtscript-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 ac7475197d9a0f3c7284f002390e2427fef84ec90dc590630431a848099c5042 qtscript-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtscript-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 9d169f447395cef5209e1e33bf1020babc035740741145705298ba81488a36be qtscript-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5sensors/qt5sensors.hash b/package/qt5/qt5sensors/qt5sensors.hash
index 4f23af0..344cc38 100644
--- a/package/qt5/qt5sensors/qt5sensors.hash
+++ b/package/qt5/qt5sensors/qt5sensors.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtsensors-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 518f392fec5e01aaa7f95e9141678cc99b02ed067ebbd24c38c1420067c59c2e qtsensors-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtsensors-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 99003fbe9304f1a135c454f28cc875f553a5f2c8c98153c516cfd8bf8ddb994d qtsensors-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5serialbus/qt5serialbus.hash b/package/qt5/qt5serialbus/qt5serialbus.hash
index da46e76..ff87c58 100644
--- a/package/qt5/qt5serialbus/qt5serialbus.hash
+++ b/package/qt5/qt5serialbus/qt5serialbus.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtserialbus-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 8cca530ee53f19672f57ffee6f2f28866d5c84b8cb3dbbd67c05d27893ac0b8e qtserialbus-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtserialbus-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 05cb306d450a697cb8311db1e0edde76eaea0b20a681a74e28cbeda2c1792452 qtserialbus-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5serialport/qt5serialport.hash b/package/qt5/qt5serialport/qt5serialport.hash
index f2f0fe9..7903d42 100644
--- a/package/qt5/qt5serialport/qt5serialport.hash
+++ b/package/qt5/qt5serialport/qt5serialport.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtserialport-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 16ee7fb66bb997df674a12a7a5a1450e2a2b860c8d34d237f3631d9f60853c1f qtserialport-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtserialport-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 a2b488fcdf076f539b601ec2c49446352d67bfe1d9fb4a57330ccc0edf5bac8c qtserialport-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5svg/qt5svg.hash b/package/qt5/qt5svg/qt5svg.hash
index 56043cf..a9df035 100644
--- a/package/qt5/qt5svg/qt5svg.hash
+++ b/package/qt5/qt5svg/qt5svg.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtsvg-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 798799319138bb40b6187f4ecc1099956a0d62148f78512e9cb085d1fa5f641f qtsvg-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtsvg-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 5c870352ecd9d0c11bfca1725999a5073c3a03e5685ba12b73aab5a81edc12f0 qtsvg-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5tools/0001-Disable-qdoc-needs-qtdeclarative.patch b/package/qt5/qt5tools/0001-Disable-qdoc-needs-qtdeclarative.patch
deleted file mode 100644
index 61cbdfe..0000000
--- a/package/qt5/qt5tools/0001-Disable-qdoc-needs-qtdeclarative.patch
+++ /dev/null
@@ -1,30 +0,0 @@
-From acdb24783322bb6e69df61cf04df2b2e47a06ad2 Mon Sep 17 00:00:00 2001
-From: Peter Seiderer <ps.report@gmx.net>
-Date: Tue, 29 Mar 2016 13:37:09 +0200
-Subject: [PATCH] Disable qdoc (needs qtdeclarative).
-
-Fixes:
-
-  Project ERROR: Unknown module(s) in QT: qmldevtools-private
-  Makefile:63: recipe for target 'sub-qdoc-qmake_all' failed
-
-Signed-off-by: Peter Seiderer <ps.report@gmx.net>
----
- src/src.pro | 1 -
- 1 file changed, 1 deletion(-)
-
-diff --git a/src/src.pro b/src/src.pro
-index 387d54f..494898f 100644
---- a/src/src.pro
-+++ b/src/src.pro
-@@ -14,7 +14,6 @@ qtHaveModule(widgets) {
- }
- 
- SUBDIRS += linguist \
--    qdoc \
-     qtplugininfo
- if(!android|android_app):!ios: SUBDIRS += qtpaths
- 
--- 
-2.1.4
-
diff --git a/package/qt5/qt5tools/qt5tools.hash b/package/qt5/qt5tools/qt5tools.hash
index 84be92d..2b43fbd 100644
--- a/package/qt5/qt5tools/qt5tools.hash
+++ b/package/qt5/qt5tools/qt5tools.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qttools-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 0d244c61bbe5505cb94310e980b06ef13dd573511e80ccbdc060f71d5462219d qttools-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qttools-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 375a528bf7141eff38cdc00c322c6d4491c8f67203fd8ee6b9a7097504605141 qttools-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5webchannel/qt5webchannel.hash b/package/qt5/qt5webchannel/qt5webchannel.hash
index 4f93cdb..bb9f0ac 100644
--- a/package/qt5/qt5webchannel/qt5webchannel.hash
+++ b/package/qt5/qt5webchannel/qt5webchannel.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtwebchannel-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 c25424935e866e77f31e5ebc50fc97eaedd4d77f6e967bfc59ce7fa6a7b4c14f qtwebchannel-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtwebchannel-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 878ab614b30bbf04ef3775af68acd47de9765973650c1db403464e79efd07e82 qtwebchannel-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5webkit/0004-Fix-linking-with-libpthread.patch b/package/qt5/qt5webkit/0004-Fix-linking-with-libpthread.patch
deleted file mode 100644
index b7b6791..0000000
--- a/package/qt5/qt5webkit/0004-Fix-linking-with-libpthread.patch
+++ /dev/null
@@ -1,34 +0,0 @@
-From 5dd4bb67cfce812fd7686e43616e2069f354a7df Mon Sep 17 00:00:00 2001
-From: Allan Sandfeld Jensen <allan.jensen@theqtcompany.com>
-Date: Mon, 22 Feb 2016 10:57:32 +0100
-Subject: [PATCH] Fix linking with libpthread
-
-WebKit use libpthread directly but is depending on other qt modules
-causing it to be linked against, which might break unless -lpthread
-is last. Instead just add it explicitly after the static libraries.
-
-Upstream-Status: Backport from 5.7 branch
-
-Change-Id: I2b95cff2c96373f8dce6f95052c4fccbe1982b33
-Reviewed-by: Simon Hausmann <simon.hausmann@theqtcompany.com>
-Signed-off-by: Jonathan Liu <net147@gmail.com>
-Signed-off-by: Gary Bisson <gary.bisson@boundarydevices.com>
----
- Tools/qmake/mkspecs/features/default_post.prf | 1 +
- 1 file changed, 1 insertion(+)
-
-diff --git a/Tools/qmake/mkspecs/features/default_post.prf b/Tools/qmake/mkspecs/features/default_post.prf
-index 67276b7..39bb3f7 100644
---- a/Tools/qmake/mkspecs/features/default_post.prf
-+++ b/Tools/qmake/mkspecs/features/default_post.prf
-@@ -201,6 +201,7 @@ needToLink() {
-         linkAgainstLibrary($$library, $$eval(WEBKIT.$${library_identifier}.root_source_dir))
-         LIBS += $$eval(WEBKIT.$${library_identifier}.dependent_libs)
-     }
-+    posix:!darwin: LIBS += -lpthread
- }
- 
- creating_module {
--- 
-2.7.1
-
diff --git a/package/qt5/qt5webkit/qt5webkit.hash b/package/qt5/qt5webkit/qt5webkit.hash
index d2ac7bb..8122cab 100644
--- a/package/qt5/qt5webkit/qt5webkit.hash
+++ b/package/qt5/qt5webkit/qt5webkit.hash
@@ -1,2 +1,2 @@
 # locally computed
-sha256 1c79843ef32313312290d4fe0b87256a9129dc44d62b01789a692981edf4d8ab  qt5webkit-d2ff5a085572b1ee24dcb42ae107063f3142d14e.tar.gz
+sha256 e4339ee069e2817fc5bf7c2c2d9fa680874da4ba56f34afdad2b4795803cbdf1 qt5webkit-b889f460280ad98c89ede179bd3b9ce9cb02002b.tar.gz
diff --git a/package/qt5/qt5webkit/qt5webkit.mk b/package/qt5/qt5webkit/qt5webkit.mk
index 629e40a..e6e8fe3 100644
--- a/package/qt5/qt5webkit/qt5webkit.mk
+++ b/package/qt5/qt5webkit/qt5webkit.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-QT5WEBKIT_VERSION = d2ff5a085572b1ee24dcb42ae107063f3142d14e
+QT5WEBKIT_VERSION = b889f460280ad98c89ede179bd3b9ce9cb02002b
 # Using GitHub since it supports downloading tarballs from random commits.
 # The http://code.qt.io/cgit/qt/qtwebkit.git/ repo doesn't allow to do so.
 QT5WEBKIT_SITE = $(call github,qtproject,qtwebkit,$(QT5WEBKIT_VERSION))
diff --git a/package/qt5/qt5websockets/qt5websockets.hash b/package/qt5/qt5websockets/qt5websockets.hash
index d803285..05abfef 100644
--- a/package/qt5/qt5websockets/qt5websockets.hash
+++ b/package/qt5/qt5websockets/qt5websockets.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtwebsockets-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 268ab869fe3d0d22abd9668851155db79aff2f386bc448051b6ef477841d719f qtwebsockets-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtwebsockets-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 34620ca34ca580802509b2fb3e9eb843d5ad11416d6a211400ac7f700aeae0bb qtwebsockets-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5x11extras/qt5x11extras.hash b/package/qt5/qt5x11extras/qt5x11extras.hash
index 5f5481e..bfec513 100644
--- a/package/qt5/qt5x11extras/qt5x11extras.hash
+++ b/package/qt5/qt5x11extras/qt5x11extras.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtx11extras-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 47ed768d02bc09edfa20332bc58c4188b9529235cdb616802ef775d6e7613425 qtx11extras-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtx11extras-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 be6cc815e9b33a20f5cd374fe77443b13ea5f1cd625616d75b3c4fdcc4f3457a qtx11extras-opensource-src-5.6.1-1.tar.xz
diff --git a/package/qt5/qt5xmlpatterns/qt5xmlpatterns.hash b/package/qt5/qt5xmlpatterns/qt5xmlpatterns.hash
index ffe2a40..68c859a 100644
--- a/package/qt5/qt5xmlpatterns/qt5xmlpatterns.hash
+++ b/package/qt5/qt5xmlpatterns/qt5xmlpatterns.hash
@@ -1,2 +1,2 @@
-# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.0/submodules/qtxmlpatterns-opensource-src-5.6.0.tar.xz.mirrorlist
-sha256 baed1b3bd3f010b8c4a96b4ca7a595b665d43d2e5758b55a364dbc9f2ac819d4 qtxmlpatterns-opensource-src-5.6.0.tar.xz
+# Hash from: http://download.qt.io/official_releases/qt/5.6/5.6.1-1/submodules/qtxmlpatterns-opensource-src-5.6.1-1.tar.xz.mirrorlist
+sha256 e1dac17a8e6cb2b03da2d666ae2a99f31c2d9b73790c79ca2c85197766cec6a4 qtxmlpatterns-opensource-src-5.6.1-1.tar.xz
-- 
cgit v0.12

